client/components/product-card/index.jsx

```diff
@@ -30,8 +30,8 @@ function ProductCard( {
 	subtitle,
 	title,
 } ) {
-	const showPriceGroup = ! subtitle && ! purchase && !! currencyCode && !! billingTimeFrame;
-	const showSecondaryHeader = subtitle || showPriceGroup;
+	const hasPriceGroup = ! purchase && !! currencyCode && !! billingTimeFrame;
+	const showSecondaryHeader = subtitle || hasPriceGroup;
```

Comment:
I would totally remove this var, but that's my personal preference only. Feel free to keep it.

Reply:
Haha, yeah, definitely! It doesn't make sense now I guess. I'm going to push this minor update as well 👍